src/find_deletions/deletion_finding_and_printing.cpp

```diff
@@ -97,22 +72,38 @@ void find_and_print_deletions(std::filesystem::path const & junction_file_path,
             {
                 if (junctions[i].get_mate1().seq_name == junctions[i].get_mate2().seq_name)
                 {
+                    variant_record tmp{};
+                    tmp.set_chrom(junctions[i].get_mate1().seq_name);
+                    tmp.set_qual(60);
+                    tmp.set_alt("<DEL>");
+                    tmp.add_info("SVTYPE", "DEL");
                     int32_t mate1_pos = junctions[i].get_mate1().position;
                     int32_t mate2_pos = junctions[i].get_mate2().position;
+                    int32_t length{};
                     if (junctions[i].get_mate1().orientation == strand::forward)
                     {
                         int32_t distance = mate2_pos - mate1_pos;
                         if (distance > 40 && distance < 100000)
                         {
-                            print_deletion(junctions[i].get_mate1().seq_name, mate1_pos, mate2_pos, 60, out_file);
+                            length = mate2_pos - mate1_pos;
```

Comment:
Why do deletions have to be dealt with separately? (compared to SNV)

Reply:
At the moment, we're just managing deletions. The plan is to have an example (deletions) to go though the whole workflow.
The other SNVs will be managed later.

Reply:
Ah, I see. Thanks for explaining :)